src/main/java/uk/gov/companieshouse/model/enums/ApplicationStatusEnum.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.model.enums;
+
+public enum ApplicationStatusEnum {
```

Comment:
Hmmm, I'm not entirely sure that I like sharing an enum with both the DB and API models. As a result they become intrinsically linked and coupled - a big no no. Although, is it overkill to create two separate enums and a mapping layer? Possibly.

Tempted to let this go for now, but something to keep an eye on.